Amasty_GdprCookie/templates/cookiebar.phtml

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * @author Amasty Team
+ * @copyright Copyright (c) 2021 Amasty (https://www.amasty.com)
+ * @package Amasty_GdprCookie
+ */
+?>
+<?php
+/** @var \Amasty\GdprCookie\Block\CookieBar $block */
+/** @var \Magento\Framework\Escaper $escaper */
+
+$notificationText = /** @noEscape */ $block->getNotificationText();
+$confirmationModal = \Amasty\GdprCookie\Model\Config\Source\CookiePolicyBarStyle::CONFIRMATION_MODAL;
+$confirmationPopup = \Amasty\GdprCookie\Model\Config\Source\CookiePolicyBarStyle::CONFIRMATION_POPUP;
+$confirmationClassic = \Amasty\GdprCookie\Model\Config\Source\CookiePolicyBarStyle::CONFIRMATION;
+?>
+<script>
+    window.isGdprCookieEnabled = true;
+</script>
+<?php
+/**
+ * Checking whether user is currenty on cookie-settings page (could be with some params)
+ * If so, block template from displaying data
+*/
+?>
+<?php if (!strpos($block->getRequest()->getRequestUri(), 'cookie-settings')): ?>
+    <?php if ($block->getNoticeType() === $confirmationModal && $notificationText): ?>
```

Comment:
good job! I think all conditions can be inside one if statement, each in a separate line:
```suggestion
<?php 
    if (
        !strpos($block->getRequest()->getRequestUri(), 'cookie-settings') &&
        $block->getNoticeType() === $confirmationModal && 
        $notificationText
    ): 
?>
```

Reply:
I must disagree with this, because below in the code we have another check for type of modal. So, combining conditions in diffrent context imo makes this repeatable (I need to add condition for page in elseif as well)

Reply:
ok, makes sense